galaxy_ng/app/urls.py

```diff
@@ -28,31 +27,31 @@
 urlpatterns.append(
     path(
         f"{API_PATH_PREFIX}/v3/openapi.json",
-        SpectacularJSONAPIView.as_view(),
+        JSONAPIView.as_view(),
         name="schema",
     )
 )
 
 urlpatterns.append(
     path(
         f"{API_PATH_PREFIX}/v3/openapi.yaml",
-        SpectacularYAMLAPIView.as_view(),
+        YAMLAPIView.as_view(),
         name="schema-yaml",
     )
 )
 
 urlpatterns.append(
     path(
         f"{API_PATH_PREFIX}/v3/redoc/",
-        SpectacularRedocView.as_view(),
+        RedocView.as_view(),
         name="schema-redoc",
     )
 )
 
 urlpatterns.append(
     path(
         f"{API_PATH_PREFIX}/v3/swagger-ui/",
-        SpectacularSwaggerView.as_view(url_name='schema'),
-        name='swagger-ui',
+        SwaggerView.as_view(url_name='schema', pulp_tag_name="Galaxy: API"),
```

Comment:
is the `pulp_tag_name` parameter needed since SwaggerView has the attribute in `galaxy_ng/app/api/views.py`?